README.md

```diff
@@ -12,44 +12,64 @@ A simple integration with Google Drive so we can upload and download files direc
 
 We created a `requirements.txt` where you can execute the following to install the needed libs:
 
-`pip install --upgrade -r requirements.txt`
-
+```sh
+pip install --upgrade -r requirements.txt
+```
 or
-
-`python -m pip install --upgrade -r requirements.txt`
+```sh
+python -m pip install --upgrade -r requirements.txt
+```
 
 ### **2. Configuring**
 
 1. Clone this repo
 
- `git clone git@github.com:DiscordTime/gdrive-manager.git`
-
+ ```sh
+ git clone git@github.com:DiscordTime/gdrive-manager.git
+ ```
  or
-
- `git clone https://github.com/DiscordTime/gdrive-manager.git`
+ ```sh
+ git clone https://github.com/DiscordTime/gdrive-manager.git
+ ```
 
  2. If the script gdrive doesn't have execute permission, give it to it.
 
- `chmod +x gdrive`
+ ```sh
+ chmod +x gdrive
+ ```
 
  3. Go to your home folder and create a directory called `.gdrive` and put your `credentials.json` file in there.
 
  4. You can add this repo folder to you path so you can execute gdrive from anywhere on your terminal. Open your `.bashrc` or similars and add one of the following options:
 
- `export PATH=$PATH:'<path-to-repo-folder>'`
+ ```sh
+ export PATH=$PATH:<path-to-repo-folder>
+ ```
+ or
+ ```sh
+ alias gdrive=<path-to-repo-folder>/gdrive
+ ```
 
- `alias gdrive='<path-to-repo-folder>/gdrive`
+**(If running on Windows):** You can create an alias for gdrive by creating a file named `gdrive.bat` on your `System32` folder with the following content:
```

Comment:
I did not suggest adding the folder to `PATH` because `gdrive` is not a file that can be run from path, only executable files (.exe, .cmd, .bat, .ps1, etc) can be run this way.
Can we skip this suggestion?

Reply:
> It does not smell good to suggest to manually alter System32 folder.

Oh, I missed that part. I can remove this section altogether if you prefer.

Reply:
An alternative would be to include a local `gdrive.bat` on this repo and add that to the `PATH`. I could do that

Reply:
Can't .py files be executed directly with the right shebang? 

https://www.python.org/dev/peps/pep-0397/#python-script-launching

https://docs.python.org/3/using/windows.html#python-launcher-for-windows

Reply:
See https://docs.python.org/3/faq/windows.html#how-do-i-make-python-scripts-executable

Reply:
As far as I know, this approach requires the source file to have `.py` extension for windows to resolve the shebang. `gdrive` has no extension

Reply:
I see. I'm willing to rename the file to `gdrive.py` and, to keep behavior on *nix, create a symbolic link named `gdrive` pointing to `gdrive.py`. For Windows, we can just run the `gdrive.py` directly, assuming the shebang logic would work, or create a `gdrive.bat` to launch the script by just running `gdrive` with the folder on PATH. Would that work for Windows?

Please test the behavior of renaming `gdrive` to `gdrive.py` and executing the file on Windows. If it works, I'll commit the rename and the symbolic link to this branch, and you can rebase your pull request.

Reply:
It seems windows will default to use pyhton2 on `.py` files when called directly, and only resolves shebangs when using `py` executable to run the file
![image](https://user-images.githubusercontent.com/5719409/108444251-9e042300-7230-11eb-89ae-ae3f12bfb312.png)
(repeated lines stand for me trying out different shebangs)

Tests using `gdrive.py`:
![image](https://user-images.githubusercontent.com/5719409/108444397-f0454400-7230-11eb-8b54-f8eda19b2075.png)
(Syntax errors are caused by trying to run the script on python2)

Then finally adding a `gdrive.bat` containing this:
```bat
@echo off
py %~dp0\gdrive.py %*
```
(`%dp0` returns batch file directory, `%*` means all arguments)
And adding repository to the `PATH` env var, worked out fine:
![image](https://user-images.githubusercontent.com/5719409/108444592-50d48100-7231-11eb-953a-fc0b35ef78cf.png)

Reply:
Thank you very much for your tests.

I have tested `gdrive.py` on Windows 10 with only a Python 3.9 installation (no Python 2). It worked fine. As Python 2 is EOL already, we should not be worrying very much about peculiarities in setups with both versions installed. Nevertheless, I think we should create a `gdrive.bat` as you described, so it will work nicely in your setup as well.

I have opened a pull request on #42 that renames the script and creates a symbolic link for *nix. Awaiting for review. Once merged, you can rebase your Pull Request.

Meanwhile, I suggest you add the following changes to your pull request:

1. Create `gdrive.bat` as you described.
2. Adjust README to reflect those new changes.

Thanks!